README.md

```diff
@@ -0,0 +1,144 @@
+# Milvus Helm Chart
+
+* For more information about installing and using Helm, see the [Helm Docs](https://helm.sh/docs/). For a quick introduction to Charts, see the [Chart Guide](https://helm.sh/docs/topics/charts/).
+
+* Installs the milvus system [Milvus installation](https://milvus.io/docs/guides/get_started/install_milvus/install_milvus.md)
+
+## Introduction
+This chart bootstraps an milvus deployment on a Kubernetes cluster using the Helm package manager.
+
+## Prerequisites
+
+- Kubernetes 1.10+
+- Helm >= 2.12.0
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+# Helm v2.x
+$ cd milvus-helm
+$ helm install --name my-release .
+```
+
+or
+
+```console
+# Helm v3.x
+$ cd milvus-helm
+$ helm install my-release  .
+```
+
+After a few minutes, you should see service statuses being written to the configured output, which is a log file inside the milvus container.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstall the Chart
+
+To uninstall/delete the my-release deployment:
+
+```console
+# Helm v2.x
+$ helm delete my-release
+```
+
+or
+
+```console
+# Helm v3.x
+$ helm uninstall my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+### Milvus Server Configuration
+
+The following table lists the configurable parameters of the milvus server and their default values.
+
+| Parameter                                 | Description                                   | Default                                                 |
+|-------------------------------------------|-----------------------------------------------|---------------------------------------------------------|
+| `primaryPath`                             | Path used to store data and metadata          | `/var/lib/milvus/data`                                  |
+| `timeZone`                                | Time zone                                     | `UTC+8`                                                 |
+| `backendURL`                              | URI format: dialect://username:password@host:port/database,Replace 'dialect' with 'mysql' or 'sqlite' | `""` |
```

Comment:
Replace "," with "." in front of "Replace".

Reply:
Done